onadata/libs/utils/middleware.py

```diff
@@ -57,21 +57,6 @@ def process_response(self, request, response):
         return response
 
 
-class BrokenClientMiddleware(MiddlewareMixin):
```

Comment:
probably not needed, but i'll test ODK Collect
https://www.flowdock.com/app/kobotoolbox/kobo/threads/X5w8Y_PgmFpwMPKR4IOVeL09mR7?filter=search&query=BrokenClientMiddleware

Reply:
added to punch list

Reply:
ODK Collect has fixed it: https://github.com/getodk/collect/pull/3269

I tested it with tcpdump and my phone set to Français and saw no evidence of localized strings in date headers :+1: